lib/membership/index.js

```diff
@@ -199,11 +199,66 @@ Membership.prototype.isPingable = function isPingable(member) {
             Member.isStatusPingable(member.status);
 };
 
+/**
+ * Change the status of the local member to alive
+ * @see Membership#setLocalStatus
+ * @see Member.Status.alive
+ */
+Membership.prototype.makeLocalAlive = function makeLocalAlive(){
+    this.setLocalStatus(Member.Status.alive);
+};
+
+/**
+ * Bump the incarnation number of the local member and return an update that
+ * described the current state of the local member.
+ *
+ * @returns {Update} The update to gossip around
+ * @private
+ */
+Membership.prototype._reincarnate = function _reincarnate() {
+    this.localMember.incarnationNumber = Date.now();
```

Comment:
I discussed this with nils and we decided to not do it right now. These PR's are refactoring stuff; not changing behaviour. If we want to move forward with the `max(inc + 1, Date.now()` we can do it at a later stage; we probably have to double check if there's not anything else breaking. 

 /cc @thanodnl